### ISSUE TITLE
XML Error when Rendering SVG

### ISSUE THREAD

**Issue by cmdvmd** [kivy/kivy#7408]
**Software Versions**
* Python: 3.9.1
* OS: Windows 10
* Kivy: 2.0.0
* Kivy installation method: pip

**Describe the bug**
When loading an SVG file, the program crashes with the following error:

```
Traceback (most recent call last):
   File "[FILE]", line 18, in <module>
     ExampleApp().run()
   File "C:\Users\[USER]\AppData\Local\Programs\Python\Python39\lib\site-packages\kivy\app.py", line 949, in run
     self._run_prepare()
   File "C:\Users\[USER]\AppData\Local\Programs\Python\Python39\lib\site-packages\kivy\app.py", line 919, in _run_prepare
     root = self.build()
   File "[FILE]", line 15, in build
     def build(self): return ExampleWidget()
   File "[FILE]", line 11, in __init__
     Svg('test.svg')  # test.svg is a file in the same directory as this program
   File "kivy\graphics\svg.pyx", line 429, in kivy.graphics.svg.Svg.__init__
   File "kivy\graphics\svg.pyx", line 519, in kivy.graphics.svg.Svg.source.__set__
   File "kivy\graphics\svg.pyx", line 535, in kivy.graphics.svg.Svg.set_tree
   File "kivy\graphics\svg.pyx", line 540, in kivy.graphics.svg.Svg.reload
   File "kivy\graphics\svg.pyx", line 571, in kivy.graphics.svg.Svg.parse_tree
 AttributeError: 'xml.etree.ElementTree.Element' object has no attribute 'getchildren'
```

**Expected behavior**
An SVG vector image is displayed on screen

**To Reproduce**

- Create a widget
- Draw an SVG with `kivy.graphics.svg.Svg` on the widget's canvas
- Start the application

**Code and Logs and screenshots**
```python
from kivy.app import App
from kivy.uix.widget import Widget
from kivy.graphics.svg import Svg


class ExampleWidget(Widget):
   def __init__(self, **kwargs):
      super().__init__(**kwargs)

      with self.canvas:
         Svg('test.svg')  # test.svg is a file in the same directory as this program


class ExampleApp(App):
   def build(self): return ExampleWidget()


ExampleApp().run()
```

**Additional context**
I have tried this program with multiple different SVG files, but ended with the same error each time.

**Comment by matham** [comment]
@tshirtman may have some insight.

**Comment by tshirtman** [comment]
I believe the code needs to be updated to directly iterate on the node, instead of using getChildren.

**Comment by tshirtman** [comment]
in fact, this was already fixed in 68cdac45c5 which has been merged in master after 2.0.0 was released, it'll be be part of 2.1.0.

**Comment by JordyScript** [comment]
@tshirtman, would it not be possible to make a 2.0.1 patch release with this fix and maybe some other bugfixes that have been merged since the last major release? It's been over half a year since the last release, and this bug has already been fixed for about the same time, but it hasn't been shipped yet because it has (unnecessarily?) been grouped together with other bugfixes and features some of which are not done yet and may not get done for a while to come.
